spring-boot-modules/spring-boot-crud/src/test/java/com/baeldung/entity_dto_differences/entity/BookUnitTest.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.entity_dto_differences.entity;
+
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class BookUnitTest {
```

Comment:
I would not typically expect to see unit tests for getters and setters. Since you already wrote them, we can keep them if you want, your choice. Typically getters and setters would already be indirectly tested via their usage in other tests.